docs/content/getting-started/setup-openfga/access-control.mdx

```diff
@@ -0,0 +1,167 @@
+   ```
+    Replace `${ACCESS_CONTROL_STORE_ID}` with the store ID you received in the previous step; replace `${FGA_ADMIN_CLIENT_ID}` with the client ID you want to grant access to.
+
+## Enable access control
+
+### Enable access control in the server
+1. Enable the experimental support for access control by setting the environment variable `OPENFGA_EXPERIMENTALS` to `enable-access-control`.
+2. Enable the access control feature by setting the environment variable `OPENFGA_ACCESS_CONTROL` to `true`.
+3. Set the environment variable `OPENFGA_ACCESS_CONTROL_STORE_ID` to the store ID you received in the previous step.
+4. Set the environment variable `OPENFGA_ACCESS_CONTROL_MODEL_ID` to the model ID you received in the previous step.
+
+### Customize what the claim you want the API to use (optional)
+
+By default, the API will use the following claims (in order) the OIDC token to identify the client. If you want to use a different claim, you can set the environment variable `OPENFGA_AUTHN_OIDC_CLIENT_ID_CLAIMS` to the claim(s) you want to use.
+
+If the claims are not set in the configuration, the following claims as default (in order):
+1. `azp`: following [the OpenID standard](https://openid.net/specs/openid-connect-core-1_0.html#IDToken)
+2. `client_id` following [RFC9068](https://www.rfc-editor.org/rfc/rfc9068.html#name-data-structure)
+
+That means that if the `azp` claim is present in the token, it will be used to identify the client. If not, the `client_id` claim will be used instead.
+
+For example, you can set the environment variable `OPENFGA_AUTHN_OIDC_CLIENT_ID_CLAIMS` to `user_id,employee_id,client_id` to allow the OpenFGA server to authorize based on:
+1. Use the `user_id` claim if present in the token.
+2. If not try to use the `employee_id` claim if present.
+3. If not try to use the `client_id` claim.
+
+## Restart the server
+
+## Grant access to a store
+
+You can now use the admin client ID to manage access to your OpenFGA server. We will call it `FGA_ADMIN_CLIENT_ID` in the following examples to differentiate it from the client ID (called `FGA_CLIENT_ID`) you are granting access to.
+
+
+We will also use `ACCESS_CONTROL_STORE_ID` as the store ID of the access control store, and `STORE_ID` as the store ID you are granting the client access to.
+
+1. Grant access to a store (based on the model above, your choices are `admin`, `model_writer`, `writer` and `reader`).
+   ```shell
+   fga tuple write --store-id "${ACCESS_CONTROL_STORE_ID}" "application:${FGA_CLIENT_ID}" model_writer "store:${STORE_ID}" --client-id "${FGA_ADMIN_CLIENT_ID}" --client-secret ... --api-token-issuer ... --api-audience ...
+   ```
+
+2. Grant access to writing tuples of a certain module in a store
+   In order to grant access only  the module feature, you must have a model with modules. Read the [modular models documentation](../../modeling/modular-models) for more.
```

Comment:
In the UI these types of paragraphs don't render properly: 
<img width="1124" alt="image" src="https://github.com/user-attachments/assets/60c6e02c-e05e-414d-a6ec-da06cb5da75c">